skglm/tests/test_penalties.py

```diff
@@ -101,5 +103,15 @@ def test_slope():
     np.testing.assert_allclose(ours.coef_, pyslope_out["beta"], rtol=1e-5)
 
 
+def test_l11():
+    ours = GeneralizedLinearEstimator(
+        datafit=QuadraticMultiTask(),
+        penalty=L1_1(alpha),
+        solver=MultiTaskBCD(tol=tol, ws_strategy="fixpoint", max_iter=10),
+    ).fit(X, Y)
+    sk = Lasso(alpha, tol=tol, fit_intercept=False).fit(X, Y)
```

Comment:
Don't you need to fit task per task?

Reply:
my bad, i wanted to compare against `Lasso`  from `sklearn`